### PR TITLE
Fix to be able to launch QuickMPC in terraform.　

### PR DESCRIPTION
**Pull request by KotaTakahashi9320**
# Summary
Fix to be able to launch QuickMPC in terraform
# Purpose
To make it easy to start up QuickMPC
# Contents
Add tf file for deploy
Fix README.md
# Testing Methods Performed
Deploy based on the README.md created
Then run demo from client

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -13,107 +13,63 @@
...
+    ...
+    variable "docker_image_tag" {
+       default = "{tag}" # {tag}を書き換える
+    }
+    ```
+3. `./terraform`ディレクトリ内にて以下のコマンドを実行する
+    ```
+    make deploy-quickmpc
+    ```
+4. GCPのネットワーク設定で{variable.tfで設定したinstance_name}-vpc-*ネットワークのファイアウォール設定を変更する. <br>
```
Comment (KotaTakahashi9320) [inline demo/README.md:29]: 4は不要
しかし、5000番ポートは空いていることをREADME.mdを追記

**Review by KotaTakahashi9320 — COMMENTED** [review]
demo/terraform/deploy→demo/terraform/application
demo/deploy→demo/hoge ex)src,deploy_files etc

instance_countはbtsも含む数とコメントを追加

File: demo/terraform/deploy/deploy.tf [hunk 2]
```diff
@@ -0,0 +1,99 @@
+locals {
+  party_num = local.instance_count - 1
+  bts_num = local.instance_count - 1
```
[the remark this answers is not in the document]
Comment (KotaTakahashi9320) [reply demo/terraform/deploy/deploy.tf]: [fa32d3f](https://github.com/acompany-develop/QuickMPC/pull/186/commits/fa32d3f2e95ad51eca88dc38f0ec465062493a2a)

File: demo/deploy/prepare_deploy.sh [hunk 3]
```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# ./prepare_deploy.sh party_num party_str image_tag
+party_num=$1
+party_str=$2
+image_tag=$3
+
+((party_num++))
```
[the remark this answers is not in the document]
Comment (KotaTakahashi9320) [reply demo/deploy/prepare_deploy.sh]: [40aaa59](https://github.com/acompany-develop/QuickMPC/pull/186/commits/40aaa5973db008337df2bb9826822528347c860f)

**Comment by KotaTakahashi9320** [comment]
@come2ry 
I have corrected the areas you reviewed.
Please check it again.

**Review by come2ry — APPROVED** [review]
LGTM!

**Pull request merged**